gpkit/nomials.py

```diff
@@ -258,13 +256,14 @@ def __le__(self, other):
         if isinstance(other, PosyArray):
             return NotImplemented
         else:
-            return Constraint(self, other)
+            return Constraint(self, other, oper_le=True)
```

Comment:
Yep, see new documentation for `Constraint.__init__` on this branch. `operable` gives full control over the left vs right and <= vs >= for users creating constraints via `Constraint.__init__`. The operator overloading way of initializing always uses `oper_le=True`.